helm/ph-ee-engine/values.yaml

```diff
@@ -705,6 +705,26 @@ operations_web:
 
 identity:
 
+
+connector_airtel:
+  enabled: false
+  replicas: 1
+  image: "docker.io/fynarfin/ph-ee-connector-airtel"
+  imageTag: "v1.6.0"
+  zeebeInitTransferWaitTimer: 10
+  tenant: ""
+  serviceAccountName: ""
+  callbackHost: ""
+  channel:
+    host: ""
+  amsMifos:
+    host: ""
+  ingress:
+    enabled: true
+    path: "/airtel"
```

Comment:
remove airtel from path. just provide '/'

Reply:
We might use Mpesa instead of Airtel based on the tenant. However, according to this [comment](https://github.com/fynarfin/ph-ee-connector-airtel/pull/1#discussion_r1670260251), such changes should be made in the channel connector. Therefore, these changes have been removed.